src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Debug/ProjectDebuggerProvider.cs

```diff
@@ -89,14 +89,10 @@ public static Guid GetManagedDebugEngineForFramework(string targetFramework)
         /// </summary>
         public static bool IsDotNetCoreFramework(string targetFramework)
         {
-            const string NetStandardAppPrefix = ".NetStandardApp";
             const string NetStandardPrefix = ".NetStandard";
             const string NetCorePrefix = ".NetCore";
-            const string NetCoreAppPrefix = ".NetCoreApp";
-            return targetFramework.StartsWith(NetCoreAppPrefix, StringComparison.OrdinalIgnoreCase) ||
-                   targetFramework.StartsWith(NetCorePrefix, StringComparison.OrdinalIgnoreCase) ||
-                   targetFramework.StartsWith(NetStandardPrefix, StringComparison.OrdinalIgnoreCase) ||
-                   targetFramework.StartsWith(NetStandardAppPrefix, StringComparison.OrdinalIgnoreCase);
+            return targetFramework.StartsWith(NetCorePrefix, StringComparison.OrdinalIgnoreCase) ||
+                   targetFramework.StartsWith(NetStandardPrefix, StringComparison.OrdinalIgnoreCase);
```

Comment:
Why is it safe to remove these checks?

Reply:
In this case:
```C#
targetFramework.StartsWith(".NetStandard", StringComparison.OrdinalIgnoreCase)
targetFramework.StartsWith(".NetStandardApp", StringComparison.OrdinalIgnoreCase)
```
If something starts with `.NetStandardApp` it will also start with `.NetStandard`.  Case 1 is a super-set of case 2. The same is true of `.NetCore` and `.NetCoreApp` right?

Reply:
Ah yes. It's clear once pointed out.